app/src/main/java/com/vitorpamplona/amethyst/ui/navigation/Routes.kt

```diff
@@ -16,52 +14,34 @@ import com.vitorpamplona.amethyst.model.Account
 import com.vitorpamplona.amethyst.ui.dal.ChatroomListKnownFeedFilter
 import com.vitorpamplona.amethyst.ui.dal.HomeNewThreadFeedFilter
 import com.vitorpamplona.amethyst.ui.dal.NotificationFeedFilter
-import com.vitorpamplona.amethyst.ui.screen.AccountStateViewModel
-import com.vitorpamplona.amethyst.ui.screen.loggedIn.AccountViewModel
-import com.vitorpamplona.amethyst.ui.screen.loggedIn.ChannelScreen
-import com.vitorpamplona.amethyst.ui.screen.loggedIn.ChatroomListScreen
-import com.vitorpamplona.amethyst.ui.screen.loggedIn.ChatroomScreen
-import com.vitorpamplona.amethyst.ui.screen.loggedIn.FiltersScreen
-import com.vitorpamplona.amethyst.ui.screen.loggedIn.HomeScreen
-import com.vitorpamplona.amethyst.ui.screen.loggedIn.NotificationScreen
-import com.vitorpamplona.amethyst.ui.screen.loggedIn.ProfileScreen
-import com.vitorpamplona.amethyst.ui.screen.loggedIn.SearchScreen
-import com.vitorpamplona.amethyst.ui.screen.loggedIn.ThreadScreen
 
 sealed class Route(
     val route: String,
     val icon: Int,
     val hasNewItems: (Account, NotificationCache, Context) -> Boolean = { _, _, _ -> false },
-    val arguments: List<NamedNavArgument> = emptyList(),
-    val buildScreen: (AccountViewModel, AccountStateViewModel, NavController) -> @Composable (NavBackStackEntry) -> Unit
```

Comment:
Removed this lambda parameter for two reasons:

1. The various `___Screen` components have different needs, and this was janky.
2. We can better manage state hoisting now in the `AppNavigation` component